.aws/.bash_aws

```diff
@@ -0,0 +1 @@
+export AWS_CONFIG_FILE=~/.aws/aws_config_file
```

Comment:
Config file is located in ~/.aws/config (default location) not sure if needed to define different location